Scriptblock/sbBrowser.ps1

```diff
@@ -0,0 +1,51 @@
+﻿
+# User browser artefact collection
+Write-Host -ForegroundColor Yellow "`tCollecting User Browser Artefacts"
+                       
+$Users = Get-ChildItem "$env:systemdrive\Users\" -Force | where-object {$_.PSIsContainer} | Where-object { $_.Name -ne "Public" -And $_.Name -ne "All Users" -And $_.Name -ne "Default" -And $_.Name -ne "Default User"} | select-object -ExpandProperty name
+
+# From https://github.com/mark-hallman/plaso_filters/blob/master/filter_windows.txt
+$ChromeFiles = @("Bookmarks", "Cookies", "Current Session", "Current Tabs", "Extension Rules", "Extension State", "Favicons", "History", "Last Session", "Last Tabs", "Preferences", "Shortcuts", "Top Sites", "Visited Links", "Web Data")
+
+Foreach ($User in $Users) {
+    ### Chrome ###
+    # Determine if user has Chrome installed
+    if ( Test-Path "$env:systemdrive\Users\$User\AppData\Local\Google\Chrome\User Data" ) {
+        # Find all Chrome profile directories
+        if ( (Test-Path variable:Triage) -and ($Triage -eq $True) ) {
+            $ChromeProfiles = @("Default")
+        } else {
+            $ChromeProfiles = Get-ChildItem "$env:systemdrive\Users\$User\AppData\Local\Google\Chrome\User Data" -recurse -include "Google Profile.ico" -force | Select-Object -Property Directory | ForEach-Object { Split-Path $_.Directory -Leaf }
+        }
+        
+        # Collect Chrome artefacts
+        foreach ($ChromeProfile in $ChromeProfiles ) {
+            $ChromeUserFiles = @()
+            $ChromeProfilePath = "$env:systemdrive\Users\$User\AppData\Local\Google\Chrome\User Data\$ChromeProfile"
+            $Destination = "$Output\User\$user\Chrome\$ChromeProfile"
+            if ( ( Test-Path variable:Triage) -and ($Triage -eq $True) ) {
+                Invoke-BulkCopy -folder $ChromeProfilePath -target $Destination -where "$_.Name -match `"(Bookmarks|History|Preferences)`""
+            } else {
```

Comment:
This extension data is all [Hindsight ](https://github.com/obsidianforensics/hindsight) needs to understand installed extensions and versions. This is better than copying the Extensions folder (even though it takes more code, it saves 15+ minutes since many extensions have tons of local files). Getting the whole extension folder could be useful in other cases if you have the time, though.